Update addon manifest docs (#527)
ScratchAddons/website-v2: content/docs/reference/addon-manifest.md

The version the addon was added. If the value is the same as the current version of the extension, the addon will get the new tag.

## `relatedAddons`

| | |
| - | - |
| Type | `Array` |

> An array of addon IDs used to link other related addons. It is not bidirectional by default.

Reviewer: ```suggestion
An array of addon IDs used to link other related addons. The relations are not bidirectional by default.

```
